Dockerfile

```diff
@@ -1,21 +1,58 @@
+# syntax=docker/dockerfile:experimental
+
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:bionic AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+ADD requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:10-slim AS yarn-dependencies
```

Comment:
We are trying to upgrade node to version 12 in all of our projects since it is the latest LTS.

Could replace `FROM node:10-slim AS yarn-dependencies` to `FROM node:12-slim AS yarn-dependencies`

Reply:
Ok, I'll do it now all with node:12 onwards.